internal/wire/pool.go

```diff
@@ -27,7 +32,20 @@ func putStreamFrame(f *StreamFrame) {
 		return
 	}
 	if protocol.ByteCount(cap(f.Data)) != protocol.MaxReceivePacketSize {
-		panic("wire.PutStreamFrame called with packet of wrong size!")
+		panic("wire.putStreamFrame called with frame of wrong size!")
+	}
+	streamFramePool.Put(f)
+}
+
+// GetAckFrame gets an ACK frame from the pool.
+// It is the callers responsibility to fill *all* of the fields of the returned ACK frame.
+func GetAckFrame() *AckFrame {
+	return ackFramePool.Get().(*AckFrame)
```

Comment:
How expensive would it be to zero out all fields here, just to be on the safe side?

(similar for the stream frame)